.php-cs-fixer.dist.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+$prestashop_config = new PrestaShop\CodingStandards\CsFixer\Config();
+$prestashop_rules = $prestashop_config->getRules();
+$prestashop_rules['blank_line_after_opening_tag'] = false;
```

Comment:
I think I would add some extra lines:

```suggestion
$prestashop_rules['blank_line_after_opening_tag'] = false;
$prestashop_rules['blank_line_before_statement'] = false;
$prestashop_rules['no_whitespace_in_blank_line'] = true;
```

But check them carefully, just in case I'm wrong